fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -487,17 +487,17 @@ private SqlQueryData processReferenceParm(Class<?> resourceType, Parameter query
      * Observation given this search parameter: device:Device.patient.family=Monella
      *
      * <pre>
-     * SELECT R.RESOURCE_ID, R.LOGICAL_RESOURCE_ID, R.VERSION_ID, R.LAST_UPDATED, R.IS_DELETED, R.DATA, LR.LOGICAL_ID FROM
-     * Observation_RESOURCES R, Observation_LOGICAL_RESOURCES LR , Observation_STR_VALUES P1 WHERE R.RESOURCE_ID = LR.CURRENT_RESOURCE_ID AND R.IS_DELETED <> 'Y' AND
-     * P1.RESOURCE_ID = R.RESOURCE_ID AND
-     * P1.PARAMETER_NAME_ID = 107 AND
-     * (p1.STR_VALUE IN
+     * SELECT R.RESOURCE_ID, R.LOGICAL_RESOURCE_ID, R.VERSION_ID, R.LAST_UPDATED, R.IS_DELETED, R.DATA, LR.LOGICAL_ID
+     * FROM Observation_LOGICAL_RESOURCES LR JOIN Observation_RESOURCES R ON R.LOGICAL_RESOURCE_ID = LR.LOGICAL_RESOURCE_ID AND R.RESOURCE_ID = LR.CURRENT_RESOURCE_ID AND R.IS_DELETED <> 'Y'
```

Comment:
minor format suggestion for sql in javadoc
```suggestion
     * FROM Observation_LOGICAL_RESOURCES LR
     * JOIN Observation_RESOURCES R ON R.LOGICAL_RESOURCE_ID = LR.LOGICAL_RESOURCE_ID AND R.RESOURCE_ID = LR.CURRENT_RESOURCE_ID AND R.IS_DELETED <> 'Y'
```

Reply:
fixed, I changed in my branch directly.